packages/backend/tsconfig.json

```diff
@@ -5,5 +5,6 @@
     "outDir": "./dist",
     "rootDir": "./src"
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*"],
+  "exclude": ["**/*.test.ts", "src/**/_*", "src/connector/core/factories/*"]
 }
```

Comment:
Any reason to exclude the tests being compiled? I also think by excluding it here the language server wont do the checking when writing tests

Reply:
There's some build errors in:
- `backend/src/connector/_logger-pino`
- `backend/src/connector/core/controllers/_ccp-protocol.ts`
- `backend/src/connector/core/services/_router`

Otherwise, there's no issues building the tests.

Is there an argument for excluding tests to reduce the build size for packaging?

Reply:
When importing the  connector, I disabled the ccp/routing code since Rafiki isn't using it (yet). I didn't delete the code since it may need it eventually. But at the moment it is dead code, and could be moved elsewhere (maybe a dummy package) or just removed.

Reply:
Its in the version control. If not being used let's kill it as it is less to maintain for now